python/pyspark/worker.py

```diff
@@ -1829,6 +1914,7 @@ def read_udfs(pickleSer, infile, eval_type):
 
     state_server_port = None
     key_schema = None
+    input_types = None
```

Comment:
nit: we can remove this as `input_types` is a local variable in some specific branches?